llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -1109,6 +1110,18 @@ void WebAssemblyTargetLowering::computeKnownBitsForTargetNode(
     }
     }
   }
+
+  // For 128-bit addition if the upper bits are all zero then it's known that
+  // the upper bits of the result will have all bits guaranteed zero except the
+  // first.
+  case WebAssemblyISD::I64_ADD128:
```

Comment:
This case triggers an "unannotated fall-through between switch labels" warning. Is it supposed to fall through from the previous case or is the previous case supposed to break?

Reply:
Oh the previous case is supposed to break, it's not supposed to fall through. Sorry about that!

I've sent https://github.com/llvm/llvm-project/pull/133783 to fix this